projects/ngrx-auto-entity/src/lib/actions/actions.ts

```diff
@@ -7,7 +7,7 @@ import { EntityAction } from './entity-action';
  * Loads a single instance of an entity, corresponding to HTTP GET /entity/:id operation
  */
 export class Load<TModel> extends EntityAction<TModel> {
-  constructor(type: new () => TModel, public keys: any, public criteria?: any, correlationId?: string) {
+  constructor(type: new () => TModel, public keys?: any, public criteria?: any, correlationId?: string) {
```

Comment:
should this be `public key?: any`? vs `keys`?

Reply:
Oh, I guess we could go either way. AE supports composite keys, which is why I used keys before. Not sure if there is any backward compatibility issue with renaming this or not...I guess people shouldn't be relying on code identifier names of any kind anyway due to the nature of minification, so renaming it would probably be fine. I guess we could do that in another story.